neuraxle/hyperparams/space.py

```diff
@@ -210,8 +211,8 @@ def with_separator(self, separator):
         return type(self)(
             separator=separator,
             **{key: value if not isinstance(value, RecursiveDict) \
-                    else value.with_separator(separator) for key, value in self.items()
-            })
+                else value.with_separator(separator) for key, value in self.items()
+               })
```

Comment:
Please use the default autoformat of pycharm using [`CTRL+ALT+L`](https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html), and always tick the "reorganize imports" box in this shortcut menu.